examples/typescript-reactflux/.gitignore

```diff
@@ -0,0 +1,69 @@
+
+/dist/
+/node_modules/
+/temp/
+/vendor/bower/
+/test/reports/
+/.tscache/
+/.grunt/
+/.tmp/
+/typings/*
+/src/.baseDir.ts
+!/Gruntfile.js
+!/typings/tsd.d.ts
+
+
+# Created by .gitignore support plugin (hsz.mobi)
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm
+
+## Directory-based project format
+.idea/
+/*.iml
+# if you remove the above rule, at least ignore user-specific stuff:
+# .idea/workspace.xml
+# .idea/tasks.xml
+# .idea/dictionaries
+# and these sensitive or high-churn files:
+# .idea/dataSources.ids
+# .idea/dataSources.xml
+# .idea/sqlDataSources.xml
+# .idea/dynamic.xml
+# and, if using gradle::
+# .idea/gradle.xml
+# .idea/libraries
+
+## File-based project format
+*.ipr
+*.iws
+
+## Additional for IntelliJ
+out/
+
+# generated by mpeltonen/sbt-idea plugin
+.idea_modules/
+
+# generated by JIRA plugin
+atlassian-ide-plugin.xml
+
+# generated by Crashlytics plugin (for Android Studio and Intellij)
+com_crashlytics_export_strings.xml
+
+
+node_modules/
+build/
+public/
+
+*.sw*
+*~
+bin/
+vendor/
+*.baseDir.ts
+*.baseDir.js
+server/app.js
+
+src/**/*.js
+src/**/*.js.map
+
+.DS_Store
+*.log
```

Comment:
Nearly all these things will need to go in ones global gitignore, not here please.